### ISSUE TITLE
Commands not working after updating to 2.3

### ISSUE THREAD

**Issue by newtinpocasideia** [ninjaknights/CosmeticMenu#13]
Basically, after i updated my PMMP, I excluded the v2.2 and put on the v2.3, the plugin loads normally, but the commands isn't working. 
I said my config yml format was outdated, I deleted it and launched the server again, didn't work at all

**Comment by Jean-TKG** [comment]
Yeah sorry but I forgot to mention that I removed the command I will add it in v3.0

**Comment by newtinpocasideia** [comment]
Oh, no problem, so... I should just wait until next release? Or the item is working by itself?

**Comment by Jean-TKG** [comment]
The item is working

**Comment by newtinpocasideia** [comment]
Can you tell me how to make the item appear on player's inventory when they spawn? I dunno if this should happen automatically in your plugin or not (sorry for bottering)

**Comment by Jean-TKG** [comment]
It should happen automatically

**Comment by Jean-TKG** [comment]
Check in the config if you add the lobby world
